addons/product/views/product_views.xml

```diff
@@ -69,14 +71,17 @@
                             <group>
                                 <group name="group_general">
                                     <field name="active" invisible="1"/>
-                                    <field name="detailed_type"/>
-                                    <field name="product_tooltip" string="" class="fst-italic text-muted" invisible="type == 'service' and not sale_ok"/>
+                                    <field name="detailed_type" widget="radio" options="{'horizontal': True}" string="Type"/>
```

Comment:
I believe we should keep the previous display until we really decrease the number of choices :D (can still easily be changed post-freeze in the worst case).